docs/manual/introduction.tex

```diff
@@ -196,7 +196,7 @@ \section{Installation\label{installation}}
 
 % Not "\url" because it looks bad in the printed manual.
 \textbf{Requirement:}
-You must have \textbf{JDK 7 or later} installed.  You can get the JDK from
+You must have \textbf{JDK 8 or later} installed.  You can get the JDK from
```

Comment:
The Checker Framework can't be built with Java 9 (See https://github.com/typetools/checker-framework/issues/1224). So, we should probably make this JDK 8.